### ISSUE TITLE
Slider Component does not announce Checked/Unchecked State

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30961]
## Description
Does not announce "checked"/"not checked" on focus of Slider Component.

This issue is covered in-depth by parent issue #30843, please check there for more details.

## React Native version:
0.63

**Comment by Simek** [comment]
@amarlette Slider component has been deprecated in the `0.59` release (https://github.com/react-native-community/releases/blob/master/CHANGELOG.md#deprecated-4) and have been extracted from the core. 

Please report the issue in this repository:
* https://github.com/callstack/react-native-slider

**Comment by amarlette** [comment]
Thank you @Simek! I have opened up issues on the repos you linked and appreciate the help with this issue.

**Comment by Simek** [comment]
@amarlette No problem, happy to help! 🙂 

FYI: You might not receive a quick response in some of the community repositories, not all of them are maintained on the daily basis, creators or maintainers involvement vary per project.

**Comment by blavalla** [comment]
While technically this would be possible, I can't imagine a UI where this would be useful or necessary. Closing out.
